doc/_static/switcher.json

```diff
@@ -0,0 +1,105 @@
+[
+    {
+        "name": "3.5.1 (stable)",
+        "version": "stable"
+    },
+    {
+        "name": "3.6 (devdocs)",
+        "version": "devdocs"
+    },
+    {
+        "version": "3.5.0"
+    },
+    {
+        "version": "3.4.3"
+    },
```

Comment:
Thoughts on just having a `3.4` option that points to the latest `3.4.x` release instead of listing all the 3.4.x releases? Or just having a `3.4.3` option?

Reply:
Thats what I did for 2.4 and older, but not here.

Reply:
Just to be clear, I'm not even sure this works.  I vote we merge it, and then fix any stylistics choices post-merge when we can see what it is like.